CK2Plus_expanded/common/artifacts/000_CK2Plus_expanded_artifacts.txt

```diff
@@ -20,4 +20,32 @@ crown_celtic_fylkir = {
 	allowed_gift = {
 		always = no
 	}
+}
+
+# Basque pagan
+#####
+
+scepter_basque_fylkir = {

```

Comment:
Why use Fylkir here? AFAIK Fylkir is a Norse worde meaning "Chief". Does it have a connection to the Basques?

Reply:
It's in line with Paradox' naming convention.